### ISSUE TITLE
Multiple cases: Expected 1 lines after "case", found 0

### ISSUE THREAD

**Issue by simPod** [slevomat/coding-standard#796]
`SlevomatCodingStandard.ControlStructures.BlockControlStructureSpacing.IncorrectLinesCountAfterControlStructure` is triggered for code like this:

```php
switch($int) {
    case 1:
    case 2:
    case 3:
        // do sth;

        break;
}
```

I think it should not require space between cases that share action.

**Comment by kukulich** [comment]
@simPod You can disable the sniff for `case`.

**Comment by simPod** [comment]
@kukulich Ah thanks, I did not realise it's just a misconfiguration
